src/main/kotlin/net/ccbluex/liquidbounce/render/buffer/AbstractFramebuffer.kt

```diff
@@ -53,7 +53,7 @@ abstract class AbstractFramebuffer: Closeable {
                 clearColor.a / 255.0F
             )
 
-            if (useDepth) {
+            if (!useDepth) {
                 glClearDepth(1.0)
             }
```

Comment:
This is wrong. The depth buffer needs to be cleared before the frame so that the old depth image does not interfere with the generation of the new image.

So it has to be cleared if there is a depth buffer. If there is no depth buffer, there is no sense in clearing it or it might even cause errors, hence the if.

Please revert this commit.